lib/theme/material_text_field_theme.dart

```diff
@@ -136,6 +137,7 @@ class FilledOrOutlinedTextTheme extends _MaterialTextFieldTheme {
     Color? suffixIconColor,
     int? helperMaxLines,
     int? errorMaxLines,
+    InputBorder? errorBorder,
```

Comment:
here `InputBorder? errorBorder` should be replaced with Color? `errorBorderColor` as we did the same for `focusedBorder` and `enabledBorder`. This makes sure that all borders (i.e. errorBorder, focusedBorder, and enabledBorder) have the same border-radius and stoke.